From 9686dd9c7670d18acff6360c8444520273d5f1b2 Mon Sep 17 00:00:00 2001
From: Jakub Wilk <jwilk@jwilk.net>
Date: Fri, 16 Jun 2023 09:24:21 +0200
Subject: [PATCH] Make regexp string raw to correct its escape sequence usage
 (GH-371)

Fixes:

    $ python3 -Wd setup.py
    setup.py:117: DeprecationWarning: invalid escape sequence \.
    ...
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 709cbc68..7a3f46e5 100644
--- a/setup.py
+++ b/setup.py
@@ -114,7 +114,7 @@ extra_options['packages'] = [
 
 def setup_extra_options():
     is_interesting_package = re.compile('^(libxml|libxslt|libexslt)$').match
-    is_interesting_header = re.compile('^(zconf|zlib|.*charset)\.h$').match
+    is_interesting_header = re.compile(r'^(zconf|zlib|.*charset)\.h$').match
 
     def extract_files(directories, pattern='*'):
         def get_files(root, dir_path, files):
-- 
2.39.2

